spec/quality_spec.rb

```diff
@@ -40,6 +40,22 @@ def check_for_extra_spaces(filename)
     "#{filename} has spaces on the EOL on lines #{failing_lines.join(", ")}"
   end
 
+  def check_for_expendable_words(filename)
+    failing_line_message = []
+    useless_words = /\b(actually|obviously|just|clearly|basically|really)\b/i
+    second_person_words = /\b(he|she)\b/i
+
+    File.readlines(filename).each_with_index do |line, number|
+      if word_found = useless_words.match(line)
+        failing_line_message << "#{filename} has '#{word_found}' on line #{number + 1}. Avoid using such words."
+      elsif word_found = second_person_words.match(line)
+        failing_line_message << "#{filename} has '#{word_found}' on line #{number + 1}. Use third person in docmentation."
```

Comment:
he/she is already third person, I think the suggestion should be something along the lines of using more generic pronouns. (i.e. 'he', 'she', and 'they' are all third person technically, but they is generic gender wise whereas 'he' and 'she' are not).

Also, 'docmentation' should be spelled 'documentation' on line 52.